### ISSUE TITLE
Make instructions for implementing new DDA formulations

### ISSUE THREAD

**Issue by GoogleCodeExporter** [adda-team/adda#106]
```
Similar to existing similar Wiki pages, it would be nice to have instructions 
to implement new:
1) polarization prescription (-pol)
2) prescription for interaction term (-int)
3) prescription for calculating scattering quantities (-scat)
4) complete DDA formulation (a combination of three above), when corresponding 
command line option will be implemented (issue 8)
```

Original issue reported on code.google.com by `yurkin` on 23 Aug 2010 at 5:13
- Blocking: #13
- Blocked on: #8

**Comment by myurkin** [comment]
Remaining parts are moved to #296, #8. And this issue is closed.
